src/web3-utils.js

```diff
@@ -219,11 +219,11 @@ export async function getMainAccount(web3) {
 }
 
 export async function getLatestBlockTimestamp(web3) {
-  const latestBlockInfo = await web3.eth.getBlock('latest')
-  if (!latestBlockInfo) {
-    return 0 // we should have a retry mechanism
+  const { timestamp } = await web3.eth.getBlock('latest')
```

Comment:
```suggestion
  const { timestamp } = await web3.eth.getBlock('latest') || {}
```

`web3.eth.getBlock()` has an annoying tendency to return `null` if no block could be returned (which should never be the case for `'latest`'... but you never know.

Reply:
Should we add the try/catch? Maybe there is a case we would like to handle the error thrown manually.

Reply:
Hmm... the only thing that would happen right now is a thrown away promise rejecting.

Let's add a debug log in `pollEvery()` and rethrow; ideally in the future we would capture this type of information in Sentry / etc.